rdt/hyper_transformer.py

```diff
@@ -188,10 +286,7 @@ def get_config(self):
                 - sdtypes: A dictionary mapping column names to their ``sdtypes``.
                 - transformers: A dictionary mapping column names to their transformer instances.
         """
-        return Config({
-            'sdtypes': self.field_sdtypes,
-            'transformers': self.field_transformers
-        })
+        return self.config
```

Comment:
One question is whether or not we should expose the actual config class to the user. Maybe we just return the dict as it used to be to them, but when they call `set_config` we create the special class.

Reply:
I think this should return a custom version of the dict (what the config class used to be) that prints nicely. I don't think we want to expose this whole class to them since it might be confusing